protos/organization_transaction.proto

```diff
@@ -0,0 +1,81 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+syntax = "proto3";
+
+import "organization.proto";
+
+message CreateOrganization {
+	
+    // The name of the organizaiton
+    string name = 1;
+
+    // Type of the organization
+    Organization.OrganizationType organization_type = 2;
+
+    // The industry of the organization
+    string industry = 3;
+
+    // Ticker symbol associated with the organization
+    string organization_id = 4;
+
+    // Four-letter code representing the identity of the organization for
+    // the purpose of providing quotes to the market
+    string pricing_source = 5;
+
+    // List of participant public keys and their roles
+    // within the organization
+    repeated Organization.Authorization authorizations = 6;
+
+    // List of the assets that the organization owns
+    repeated Organization.Holding holdings = 7;
```

Comment:
Holdings are created in their own transaction, so we wont need a holdings field in the CreateOrganization txn